_rest_routes.inc.php

```diff
@@ -372,10 +373,20 @@
         $data = (array)(json_decode(file_get_contents("php://input"), true));
         return (new FhirPatientRestController(null))->put($pid, $data);
     },
+    "PATCH /fhir/Patient/:pid" => function ($pid) {
+        RestConfig::authorization_check("patients", "demo");
+        $data = (array)(json_decode(file_get_contents("php://input"), true));
+        return (new FhirPatientRestController(null))->put($pid, $data);
+    },
     "GET /fhir/Organization" => function () {
         return (new FhirOrganizationRestController(null))->getAll($_GET);
     },
     "GET /fhir/Organization/:oid" => function ($oid) {
         return (new FhirOrganizationRestController(null))->getOne($oid);
+    },
+    "POST /fhir/QuestionnaireResponse" => function () {
+        RestConfig::authorization_check("patients", "demo");
+        $data = (array)(json_decode(file_get_contents("php://input"), true));
+        return (new FhirQuestionnaireResponseController(null))->post($data);
```

Comment:
We haven't introduced FhirQuestionnaireResponseController in the codebase 😕

Reply:
If you guys would give me longer than a minute, we just may in day or two.:) Kidding, i'm working it. It's a resource that will help us answer many questions...